### DISCUSSION TITLE
The photo failed to upload.

### DISCUSSION THREAD

**Discussion by pablozagni** [andrewdwallo/filament-companies#117] [Q&A] [answered]
In /user/profile I get the error "The photo failed to upload." Storage Link is created.
I've also created c:\laragon\www\erpsaas\storage\app\public\profile-photos and still get the error.
Thanks

**Comment by andrewdwallo** [comment]
Can you show me the full stacktrace? What version are you on? Was the upload size too large? Mine works fine.

**Comment by pablozagni** [comment, marked as answer]
Sorry. It was a PHP version Issue... I've upgrade to 8.3 and it stores uploads in a folder with no rights. Thanks!
